components/NavBar/index.tsx

```diff
@@ -13,16 +15,18 @@ interface NavbarProps {
 }
 
 const Navbar = ({ links }: NavbarProps) => {
-  const [signIn, setSignIn] = useState(false)
-  const toggleSignIn = useCallback(() => {
-    setSignIn(prev => !prev)
+  const [openSignInForm, setOpenSignInForm] = useState(false)
+  const { user } = useFirebase()
+
+  const toggleOpenSignInForm = useCallback(() => {
+    setOpenSignInForm(prev => !prev)
   }, [])
 
   return (
     <header className='fixed inset-x-0 top-0 z-10 hidden py-2 bg-primary md:block'>
-      <nav className='font-serif text-2xl text-white container flex justify-between px-3 mx-auto'>
+      <nav className='container flex justify-between px-3 mx-auto font-serif text-2xl text-white'>
         <Link href='/'>
-          <a className='flex relative hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus: focus:ring-opacity-50 rounded'>
+          <a className='relative flex rounded hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus: focus:ring-opacity-50'>
```

Comment:
dangling classes declared
```suggestion
          <a className='relative flex rounded hover:opacity-75 focus:outline-none focus:ring focus:ring-teal/50'>
```